lib/matplotlib/tests/test_dates.py

```diff
@@ -243,11 +243,11 @@ def _create_auto_date_locator(date1, date2):
                  '1990-09-01 00:00:00+00:00', '1990-10-01 00:00:00+00:00',
                  '1990-11-01 00:00:00+00:00', '1990-12-01 00:00:00+00:00']
                 ],
-               [datetime.timedelta(days=140),
-                ['1990-01-06 00:00:00+00:00', '1990-01-27 00:00:00+00:00',
-                 '1990-02-17 00:00:00+00:00', '1990-03-10 00:00:00+00:00',
-                 '1990-03-31 00:00:00+00:00', '1990-04-21 00:00:00+00:00',
-                 '1990-05-12 00:00:00+00:00']
+               [datetime.timedelta(days=141),
```

Comment:
@tacaswell It's an expected (minor) behavior change caused by an increase in the accuracy of the way marker locations are calculated. I think the change was in [this commit](https://github.com/pganssle/matplotlib/commit/2f809168fb601ff9b89f07d9700c0073248d64b1); previously if the beginning and end dates were January 3rd, 2000 and June 10th, 2015, `relativedelta` object returns `(years=15, months=5, days=7)`, and when `get_locator()` is called, it calculated the number of days between the dates assuming that the number of days per year is 365 and the number of days per month is 30, so the number of days calculated would be 5632 rather than the true value of 5637.

It's a small inaccuracy, but it's easy to fix by holding on to a `timedelta` object and using that when calculating days from years or months. As a result, some of the date range calculations are slightly different (though not to a degree that anyone would really notice), so at least one of the tests had to be updated (I vaguely remember updating one that was around 3 months and one that was around 3 weeks).

(Oops, responded to this in the PR not the line)